src/main/java/de/cronn/jira/sync/domain/JiraFieldsUpdate.java

```diff
@@ -79,6 +80,12 @@ public void setVersions(Set<JiraVersion> versions) {
 		this.versions = versions;
 	}
 
+	@JsonIgnore
```

Comment:
shouldn’t be necessary for a `with…` method

Reply:
Removed.